packages/superset-ui-control-utils/src/index.ts

```diff
@@ -2,16 +2,20 @@ import * as constantsModule from './constants';
 import * as sharedControlsModule from './shared-controls';
 import * as sectionModules from './sections';
 
+// explore all available shared controls
+export { default as sharedControls } from './shared-controls';
```

Comment:
THANK YOU. I happen to be in need of this presently, so this has my full support :D